packages/xml-views-quick-fix/test/quick-fix-stable-id-spec.ts

```diff
@@ -16,19 +18,59 @@ describe("the UI5 language assistant QuickFix Service", () => {
                                   </mvc:content>
                               </mvc:View>`;
       const expectedSuggestion = ' id="_IDGenList1"';
-      const { document, start, end, idStartOffest } = getXmlSnippet(
+      const { document, quickFixStableIdTestInfo } = getXmlSnippet(
         testXmlSnippet
       );
+      expect(quickFixStableIdTestInfo).to.not.be.empty;
       const testXmlDoc = getXmlDocument(document);
       const quickFixInfo = computeQuickFixStableIdInfo(testXmlDoc, [
         {
-          start,
-          end,
+          start: quickFixStableIdTestInfo[0].start,
+          end: quickFixStableIdTestInfo[0].end,
         },
       ]);
       expectExists(quickFixInfo, "Quick Fix Info");
       expect(quickFixInfo[0].newText).to.equal(expectedSuggestion);
-      expect(quickFixInfo[0].replaceRange.start).to.equal(idStartOffest);
+      expect(quickFixInfo[0].replaceRange.start).to.equal(
+        quickFixStableIdTestInfo[0].idStartOffest
+      );
+    });
+
+    it("will get quick fix suggestions when classes are missing id attribute key", () => {
+      const testXmlSnippet = `<mvc:View
+                                xmlns:mvc="sap.ui.core.mvc"
+                                xmlns="sap.m"> 
+                                  <mvc:content>
+                                    <🢂List🢀$></List>
+                                    <🢂List🢀$></List>
+                                  </mvc:content>
+                              </mvc:View>`;
+      const expectedFirstSuggestion = ' id="_IDGenList1"';
```

Comment:
we do not need these consts.
They actually reduce readability of the test, because, when a developer **reads** the test they will need to resolve the reference and scroll up to understand what is actually expected.

`      expect(quickFixInfo[0].newText).to.equal(expectedFirstSuggestion);`